### PR TITLE
Feature/victor/mistral

### PR DESCRIPTION
**Pull request by TheVicBro**
Mistral Implementation using the same current formatting as OpenAI

### CODE REVIEW

**Comment by laxitshahi** [comment]
Waiting on model type discussion before merge

**Review by laxitshahi — CHANGES_REQUESTED** [review]
As discussed in the meeting today, you should implement the chatbot version of mistral so that we can get responses rather than completions

File: src/models/mistral.py
Comment (laxitshahi) [inline src/models/mistral.py:19]: As discussed in the meeting today, you should implement the chatbot version of mistral so that we can get responses rather than completions

**Review by laxitshahi — CHANGES_REQUESTED** [review]

File: src/models/mistral.py
Comment (TheVicBro) [reply src/models/mistral.py]: As mentioned, I switched the API to Mistral-Instruct

File: llmproxy/models/mistral.py
Comment (laxitshahi) [inline llmproxy/models/mistral.py:40]: The payload should return data if there is no error, i.e. when there is an error you would return a completionresponse with no payload, err with error type, and message with the error

**Review by laxitshahi — CHANGES_REQUESTED** [review]

**Review by laxitshahi — APPROVED** [review]

**Review by dinuja9 — APPROVED** [review]

**Pull request merged**
